travis/travis_run_tests

```diff
@@ -37,6 +38,9 @@ if __name__ == '__main__':
     if os.environ.get("LINT_CHECK", "1") == "1":
         tests.append(['test_flake8'])
         tests.append(['test_pylint'])
-    if os.environ.get("LINT_CHECK", None) != "1":
+    # Execute tests by default. Disable them on lint checks
+    if (os.environ.get("LINT_CHECK") != "1" and
```

Comment:
if LINT_CHECK is is not set, `.get()` returns `None` which is different than `"1"`, so it is not needed to default to `'0'`.